Section 1- Python Crash Course/2-Python List,Dictionary,Sets etc.ipynb

```diff
@@ -164,7 +164,7 @@
     }
    ],
    "source": [
-    "print(my_str.isalnum()) #check if all char are numbers\n",
+    "print(my_str.isalnum()) #check if all char are numbers or alphabets\n",
```

Comment:
```suggestion
    "print(my_str.isalnum()) #check if all char are numbers or alphabets\n",
```
You should put this `alphanumeric` instead of putting both alphabets and numbers!
Thanks